src/main/java/uk/ac/cam/cl/dtg/segue/dao/users/PgUsers.java

```diff
@@ -755,9 +755,10 @@ private RegisteredUser createUser(final RegisteredUser userToCreate) throws Segu
             setValueHelper(pst, 12, userToCreate.getLastSeen());
```

Comment:
while you're in here, it might be worth converting these integer indices into named constants to make the code a bit clearer (and to stop checkstyle whingeing)

Reply:
This is changed in another ticket that Robert is working on (The checkstyle ticket)